src/planning/planning/utils/path_3d_rotation.py

```diff
@@ -1,55 +1,196 @@
+import matplotlib
+matplotlib.use('Agg')
+
 import numpy as np
-from scipy.interpolate import make_interp_spline
-from scipy.spatial.transform import Rotation, RotationSpline
+from scipy.interpolate import make_interp_spline, make_lsq_spline
+from scipy.spatial.transform import Rotation#, RotationSpline
```

Comment:
Can we remove this RotationSpline comment?

Reply:
Cleaned up imports overall